robottelo/test.py

```diff
@@ -132,13 +134,26 @@ class UITestCase(TestCase):
     def setUpClass(cls):  # noqa
         """Make sure that we only read configuration values once."""
         super(UITestCase, cls).setUpClass()
-        cls.katello_user = settings.server.admin_username
-        cls.katello_passwd = settings.server.admin_password
+
+        cls.katello_default_user = settings.server.admin_username
```

Comment:
I don't see this being used anywhere, we can drop this. Or am I missing something?